vocables.js

```diff
@@ -91,6 +91,137 @@ const vocables = (german, norsk) => [
   { [german]: ['Wie geht es dir?'], [norsk]: ['Hvordan har du det?', 'Hvordan går det?'] },
   { [german]: ['Danke, mir geht es gut.', 'Danke, ganz gut.'], [norsk]: ['Takk, ganske bra.'] },
 
+  // Voc. Lection 2
+  { [german]: ['bedeuten'], [norsk]: ['bety'] },
+  { [german]: ['eine Sahnetorte'], [norsk]: ['ei bløtkake'] },
+  { [german]: ['ein Tisch'], [norsk]: [''et bord] },
```

Comment:
Hier gibt es einen Fail bei `''et bord`.

Guck mal in die [README](https://github.com/oleblaesing/norsk/blob/master/README.md), da steht drin wie du die App lokal starten kannst, umzu schauen, ob deine Änderungen Fehler verursachen.